spec/puppet-syntax/templates_spec.rb

```diff
@@ -4,7 +4,8 @@
   let(:subject) { PuppetSyntax::Templates.new }
   let(:conditional_warning_regex) do
     if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.6.0')
```

Comment:
Perhaps a good excuse to change this to drop Ruby < 2.6 compatibility?

Reply:
done